app/services/pdf/TADPdfGenerator.scala

```diff
@@ -41,7 +41,7 @@ class TADPdfGenerator @Inject() (
 
   def generateP5TAD(ie029Data: IE029Data): Array[Byte] = {
 
-    val renderedDocument = documentP5.render(ie029Data)
+    val renderedDocument = documentP5.render(ie029Data, TableViewModel()(ie029Data))
```

Comment:
Can we just pass the viewModel here? Rather than the viewModel and the IE029Data model